### PR TITLE
Add Windows to GitHub test job

### LINKED ISSUE #101: Add GH actions job with runs-on: windows-latest
In order to ensure that all tests work in Windows, it will be very useful to add to GH Actions this job.

### PR DESCRIPTION
**Pull request by vincentducorps**
Resolve #101 
Related #99

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -3,19 +3,20 @@ on: [pull_request]
...
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Install dependences
+      - name: Install dependencies
         run: |
-          npm install -g yarn 
+          npm install -g yarn
           sudo apt-get install tree
```
Comment (vincentducorps) [inline .github/workflows/test.yml]: My bad, I forget this line...

Comment (aralroca) [inline .github/workflows/test.yml]: I guess that now we can remove this line because we are not using `tree` any more 👍

Comment (vincentducorps) [reply .github/workflows/test.yml]: Fixed: https://github.com/vinissimus/next-translate/pull/103/commits/5551a763c570ed8641787f8ccdd7a042d3089879

Comment (vincentducorps) [reply .github/workflows/test.yml]: Fixed: https://github.com/vinissimus/next-translate/pull/103/commits/5551a763c570ed8641787f8ccdd7a042d3089879

**Pull request merged**